x-pack/test/fleet_api_integration/config.ts

```diff
@@ -15,7 +15,8 @@ import { defineDockerServersConfig } from '@kbn/test';
 // example: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fpackage-storage/detail/snapshot/74/pipeline/257#step-302-log-1.
 // It should be updated any time there is a new Docker image published for the Snapshot Distribution of the Package Registry.
 export const dockerImage =
-  'docker.elastic.co/package-registry/distribution:b6a53ac9300333a4a45f3f7d350c9aed72061a66';
+  process.env.FLEET_PACKAGE_REGISTRY_DOCKER_IMAGE ||
+  'docker.elastic.co/package-registry/distribution:staging';
```

Comment:
We should keep using a specific commit id to make sure CI stays stable. If you want to get a more recent distribution version, just take a commit hash. Each should have a docker image. The `staging` one changes every day.

Reply:
It's also running tests on a package which is months old and no longer published. What confidence do we gain from that?

I have a separate Draft PR at https://github.com/elastic/kibana/pull/101911. Let's discuss this there.